federation/cmd/federation-apiserver/app/server.go

```diff
@@ -87,6 +87,8 @@ func Run(s *options.ServerRunOptions, stopCh <-chan struct{}) error {
 func NonBlockingRun(s *options.ServerRunOptions, stopCh <-chan struct{}) error {
 	// register all admission plugins
 	registerAllAdmissionPlugins(s.Admission.Plugins)
+	genericapiserver.RegisterAllAdmissionPlugins(s.Admission.Plugins)
```

Comment:
What do you think about having this call happen when the `AdmissionOptions` are created?  I think its pretty reasonable to register "special" plugins the base namespace.  We can make everyone else namespace theirs to avoid collisions, but the core ones should all be common.